src/index.js

```diff
@@ -83,10 +84,20 @@ class App extends React.Component {
     this.setState({ [event.target.name]: event.target.value });
   }
 
+  handleExportFormatChange = (event) => {
+    console.log(event.target.name, event.target.value)
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  exportTranscript = (exportFormat) => {
+    const {data, ext} = this.refs.transcriptEditor.exportData(this.state.exportFormat);
+    this.download(data, `${ this.state.mediaUrl } .${ext}`);
```

Comment:
same here, I'd remove the space between file name and file extension

```suggestion
    this.download(data, `${ this.state.mediaUrl }.${ext}`);
```